src/Approximations/overapproximate.jl

```diff
@@ -1032,6 +1033,9 @@ function overapproximate(vTM::Vector{TaylorModel1{T, S}},
 
     # compute overapproximation
     return _overapproximate_vTM_zonotope!(vTM, c, gen_lin, gen_rem)
+
+    # remove generators which are zero
+    return remove_zero_generators(Z)
```

Comment:
```suggestion
    Z = _overapproximate_vTM_zonotope!(vTM, c, gen_lin, gen_rem)

    # remove generators which are zero
    return remove_zero_generators(Z)
```

Reply:
this part, is still broken i have to think about how to fix it..

Reply:
finally, i fixed the zero gens issue in the main loop. i dont want to close https://github.com/JuliaReach/LazySets.jl/issues/2106 yet, to give it a second thought later